lighthouse-core/report/report-generator.js

```diff
@@ -173,7 +173,8 @@ class ReportGenerator {
    * @return {string}
    */
   getReportJS() {
-    return fs.readFileSync(path.join(__dirname, './scripts/lighthouse-report.js'), 'utf8');
+    const scriptSrc = fs.readFileSync(path.join(__dirname, './scripts/lighthouse-report.js'), 'utf8');
+    return `<script>${scriptSrc}</script>`;
```

Comment:
can we keep `getReportJS()` the same and use handlebars to conditionally write them if there's a `script`?